extension/encoding/googlecloudlogentryencodingextension/internal/passthroughnlb/passthroughnlb-events.md

```diff
@@ -0,0 +1,38 @@
+# Semantic conventions for GCP Passthrough Network Load Balancer log events
```

Comment:
@open-telemetry/semconv-log-approvers Is this what you had in mind for sem conv close to the implementation, as we discussed in the meeting on Tuesday 9 December?